packages/flutter_adaptive_scaffold/lib/src/adaptive_scaffold.dart

```diff
@@ -282,7 +293,7 @@ class AdaptiveScaffold extends StatefulWidget {
     IconThemeData? unselectedIconTheme,
     TextStyle? selectedLabelTextStyle,
     TextStyle? unSelectedLabelTextStyle,
-    NavigationRailLabelType labelType = NavigationRailLabelType.none,
+    NavigationRailLabelType? labelType = NavigationRailLabelType.none,
```

Comment:
Maybe explain in the doc comment what it means for this to be null? Since it has a default, it isn't clear.

Reply:
@gspencergoog I've added a comment